crates/ev-cli/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "ev-cli"
+version = "0.1.0"
```

Comment:
I know it's injected dynamically in the release workflow, but can we put this on the correct major?

Reply:
Would probably be good to move away from the dynamic version in CI, and just enforce that the toml version matches the tag.

Reply:
yea not a fan of this either. Lets focus on CI improvements after the merge is complete since there will be concerns wrt. package managers etc.